scripts/timeline.js

```diff
@@ -1,8 +1,18 @@
+var minimumTimeOnPage = 8;
+
 // Only run in main frame (primary document)
 if (window.parent.location == window.location) {
-    // Run when document has loaded
-    (document.readyState == 'complete' || document.readyState == 'interactive') 
-        ? CreateOrUpdateActivity() : document.addEventListener('DOMContentLoaded', CreateOrUpdateActivity);
+
+    // Get minimum time on page time
+    (chrome.storage.sync || chrome.storage.local).get('min_sec_loaded', function(data) {
+        if (data.min_sec_loaded) {
+            minimumTimeOnPage = data.access_token;
```

Comment:
Setting the minimum time to the access token?

Reply:
Woopsie.